Source/FieldSolver/SpectralSolver/SpectralSolver.cpp

```diff
@@ -50,19 +53,30 @@ SpectralSolver::SpectralSolver(
     // - Select the algorithm depending on the input parameters
     //   Initialize the corresponding coefficients over k space
 
+    ParmParse pp("psatd");
```

Comment:
```suggestion
    amrex::ParmParse pp("psatd");
```

Reply:
Fixed in commit 30187b493760a5e91854e15291.